### PR TITLE
Update M.E.AI code coverage mins from 0

### PR DESCRIPTION
**Pull request by stephentoub**
Based on https://github.com/dotnet/extensions/pull/5640#issuecomment-2499318616
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/extensions/pull/5698)

### CODE REVIEW

File: src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <PropertyGroup>
     <Stage>preview</Stage>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
-    <MinCodeCoverage>0</MinCodeCoverage>
+    <MinCodeCoverage>80</MinCodeCoverage>
```
Comment (RussKie) [inline src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj:12]:
```suggestion
    <MinCodeCoverage>83</MinCodeCoverage>
```

File: src/Libraries/Microsoft.Extensions.AI.AzureAIInference/Microsoft.Extensions.AI.AzureAIInference.csproj [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <PropertyGroup>
     <Stage>preview</Stage>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
-    <MinCodeCoverage>0</MinCodeCoverage>
+    <MinCodeCoverage>70</MinCodeCoverage>
```
Comment (RussKie) [inline src/Libraries/Microsoft.Extensions.AI.AzureAIInference/Microsoft.Extensions.AI.AzureAIInference.csproj:12]:
```suggestion
    <MinCodeCoverage>77</MinCodeCoverage>
```

File: src/Libraries/Microsoft.Extensions.AI.Ollama/Microsoft.Extensions.AI.Ollama.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
   <PropertyGroup>
     <Stage>preview</Stage>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
-    <MinCodeCoverage>0</MinCodeCoverage>
+    <MinCodeCoverage>75</MinCodeCoverage>
```
Comment (RussKie) [inline src/Libraries/Microsoft.Extensions.AI.Ollama/Microsoft.Extensions.AI.Ollama.csproj:12]:
```suggestion
    <MinCodeCoverage>80</MinCodeCoverage>
```

File: src/Libraries/Microsoft.Extensions.AI.OpenAI/Microsoft.Extensions.AI.OpenAI.csproj [hunk 4]
```diff
@@ -9,7 +9,7 @@
   <PropertyGroup>
     <Stage>preview</Stage>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
-    <MinCodeCoverage>0</MinCodeCoverage>
+    <MinCodeCoverage>60</MinCodeCoverage>
```
Comment (RussKie) [inline src/Libraries/Microsoft.Extensions.AI.OpenAI/Microsoft.Extensions.AI.OpenAI.csproj:12]:
```suggestion
    <MinCodeCoverage>66</MinCodeCoverage>
```

File: src/Libraries/Microsoft.Extensions.AI/Microsoft.Extensions.AI.csproj [hunk 5]
```diff
@@ -11,7 +11,7 @@
   <PropertyGroup>
     <Stage>preview</Stage>
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
-    <MinCodeCoverage>0</MinCodeCoverage>
+    <MinCodeCoverage>80</MinCodeCoverage>
```
Comment (RussKie) [inline src/Libraries/Microsoft.Extensions.AI/Microsoft.Extensions.AI.csproj:14]:
```suggestion
    <MinCodeCoverage>83</MinCodeCoverage>
```

**Review by RussKie — APPROVED** [review]

**Comment by stephentoub** [comment]
We want the limits to be this tightly coupled to the current coverage? It breeds zero breathing room. Does the tooling fail CI if the coverage drops below the minimum?

**Comment by RussKie** [comment]
> 🎉 **Good job! The coverage increased** 🎉 Update `MinCodeCoverage` in the project files.
> 
> Project	Expected	Actual
> Microsoft.Extensions.Diagnostics.Probes	70	**76**
> Microsoft.Extensions.Caching.Hybrid	75	**84**
> Full code coverage report: [dev.azure.com/dnceng-public/public/_build/results?buildId=880346&view=codecoverage-tab](https://dev.azure.com/dnceng-public/public/_build/results?buildId=880346&view=codecoverage-tab)

https://github.com/dotnet/extensions/pull/5700 to address the remaining

**Pull request merged**